### ISSUE TITLE
Twenty Twenty-One: No global style option on Atomic Sites

### ISSUE THREAD

**Issue by ezebecke** [Automattic/themes#3202]
## Summary

Twenty Twenty-One has no option to change Global Styles. If we have changed the custom font while using a different block based theme, and now we want to change the font to the default theme font we need to change the theme, reset the font to default, and change our theme back to Twenty Twenty-One.

UPDATE: This issue is limited to Atomic Sites.

## Steps to replicate

1- Go to any page editor while using Twenty Twenty-One

## Result

Global styles option won't appear. 

## Expected

To see the Global Styles option like any other block based theme. 

![Markup 2021-02-06 at 12 47 38](https://user-images.githubusercontent.com/24919469/107127117-954b3e80-6879-11eb-9ae9-378dced68871.png)

* User report: #3712817-zd

**Comment by donalirl** [comment]
Encountered in #18659345-hc as well

**Comment by supernovia** [comment]
Adding to this, a user would like to change font pairings after changing themes, but can't because of this issue.
https://wordpress.com/forums/topic/changing-font-pairing/

I thought it might be some kind of regression of this:
https://github.com/Automattic/wp-calypso/pull/40652

But really if they could just access global styles (or even Fonts under Customize) they'd be okay.

**Comment by msilbers** [comment]
issue reported in 3759469-zd

The customer is understandably upset that this theme has no font options, whether global styles or Customizer Fonts.

**Comment by rinazrina** [comment]
Another case: #3801920-zen

**Comment by msilbers** [comment]
reported by a customer in 3802882-zen

**Comment by aleone89** [comment]
Hello, this has been reported in the forums here: https://wordpress.com/forums/topic/fonts-115/

**Comment by scosgro** [comment]
Another report in 28656961hc

**Comment by kavyagokul** [comment]
Another one in 28953679-hc

**Comment by zdenys** [comment]
1080226-hc

**Comment by geekinthegirl** [comment]
Adding another report here: 29195287-hc

**Comment by pbking** [comment]
Welp.
I added the support to enable global styles: 
```
add_theme_support( 'jetpack-global-styles' );
```

But what I see (on my sandbox) is this:
![image](https://user-images.githubusercontent.com/146530/120368867-93056800-c2e0-11eb-8515-964dd8ee96dc.png)

Something seems to be getting buggered up in [the plugin](https://github.com/automattic/wp-calypso/tree/trunk/apps/editing-toolkit).  So I got said plugin running locally with the theme and of course it works OK running everything locally.

So I'm a bit stuck!  The theme is setup to handle the globally selected fonts, you can't can set them while using the theme.  Which is certainly a problem.  I just can't get to the bottom of it!

**Comment by pbking** [comment]
Further investigation informs me that... strangely and for reasons I can't yet explain... the directory 'twentytwentyone' seems to have some magical power to break it per the above.

I copied the theme (as modified noted above) to another folder and activated it and ... it worked perfectly.

**Comment by pbking** [comment]
Further research shows that when the global-styles plugin is making the call to determine what to show it results in an error when using the twentrytwentyone theme:

```
https://public-api.wordpress.com/wp/v2/sites/183771025/jetpack-global-styles?_envelope=1&environment-id=stage&_gutenberg_nonce=4d38073dfc&_locale=user

{"body":{"code":"rest_forbidden","message":"Sorry, you are not allowed to do that.","data":{"status":403}},"status":403,"headers":[]}
```

**Comment by pbking** [comment]
Aaaaaand all of the above was because I wasn't sandboxing the api endpoints.

**Comment by pbking** [comment]
Change can be found: c-D62156

**Comment by jffng** [comment]
Tested and verified the above patch adds the ability to change the heading and base font for twentytwentyone.

**Comment by kathrynwp** [comment]
@scruffian Thanks for fixing this issue! I can't seem to find the Global Styles option on an Atomic site running Twenty Twenty One, has this change been merged into Atomic sites running this theme? Was able to replicate on an Atomic test site.

User in 4343666-zen would need to be notified. I will suggest they use a Google Fonts plugin in the meantime.

**Comment by scruffian** [comment]
This fix should also be there on Atomic, so I'm not sure why the user can't see it, sorry.

**Comment by jeffikus** [comment]
@scruffian yes, is this a problem between the version that's loaded on Atomic sites and on wpcom perhaps?

**Comment by jordesign** [comment]
Re-testing this in triage and can confirm that Global Styles are available on Simple Sites, but not Atomic sites. Would be great to prioritise this.

**Comment by scruffian** [comment]
I just tested on my Atomic site https://scruffianbusiness.wpcomstaging.com/ and it seems to be there:
<img width="1440" alt="Screenshot 2021-12-13 at 09 59 09" src="https://user-images.githubusercontent.com/275961/145791429-7634444c-9e4e-4b5d-817c-9f23f3b02211.png">

**Comment by kriskorn** [comment]
@scruffian The only difference I saw ( which could make any difference ) on your test site and on my test site - https://123kristraining456.wpcomstaging.com/ that you have something called `editing-toolkit` active, which Atomic sites do not normally have. I have `full-site-editing` active, but that does not make a difference. 

<img width="1719" alt="Screenshot 2021-12-13 at 12 52 05" src="https://user-images.githubusercontent.com/12064241/145799593-c3444a97-3d79-41a0-8635-46098a4280bd.png">

**Comment by scruffian** [comment]
Do you see global styles on other themes on test sites that do not have the editing toolkit active?

**Comment by kriskorn** [comment]
It did not show for Quadrat, but showed for Seedlet and Maywood.

**Comment by scruffian** [comment]
Is that on the same site? The behaviour I am seeing is the same for all themes:
- If the editor toolkit plugin is enabled then Jetpack Global Styles shows
- If the editor toolkit plugin is disabled then Jetpack Global Styles does not show

**Comment by kriskorn** [comment]
Yes, that is on the same site. I think the issue here is that `editing-toolkit` is not a plugin that is installed on Atomic by default. However, `full-site-editing` is. If `editing-toolkit` is a plugin that is needed on Atomic for Global Styles to work, though I got mixed results, then this would have to be added to the plugin list that needs to be installed and activated on all Atomic sites by default.

**Comment by scruffian** [comment]
@Automattic/cylon Do you know anything about this?

**Comment by creativecoder** [comment]
`full-site-editing` is still the [plugin slug for the Editing Toolkit Plugin](https://wordpress.org/plugins/full-site-editing/). There should be an up to date version of the plugin on every Atomic site, but it may not be activated, particularly for older sites.

<img width="954" alt="image" src="https://user-images.githubusercontent.com/1699996/146621030-bc870163-63cb-4bdd-b5cf-7c6e4ad3df72.png">

I believe the Jetpack Global styles functionality still lives in that plugin, so if it's missing, activating the plugin might fix it.

**Comment by scruffian** [comment]
I think we can close this. If a site doesn't have Jetpack Global Styles you can enable it by enabling the WordPress.com Editing Toolkit plugin.

**Comment by kriskorn** [comment]
@scruffian I think I may not have been very clear above. 

I have just reset my Atomic test site. It has Twenty Twenty One symlinked and all the default plugins installed. The **WordPress.com Editing Toolkit** is installed, but that does not do anything. This plugin has a slug of `full-site-editing`.

However, on your test site, I can see a plugin with a slug of `editing-toolkit` is active, but `full-site-editing` plugin is inactive on your test site. As far as I can see, this is the plugin that provides you with the Global Styles option on your site, but this is not a plugin that is installed on Atomic sites by default. Could you please check what is the actual name of the plugin in WP Admin -> Plugins page and/or deactivate it and activate the **WordPress.com Editing Toolkit** plugin instead to see if Global Styles are still shown to you? 

I have also sent you an invite to my test site, so you could take a look that no Global Styles are shown for Twenty Twenty One with default plugins installed.
